elyra/metadata/metadata.py

```diff
@@ -169,8 +171,8 @@ class MetadataStore(ABC):
     def __init__(self, namespace, **kwargs):
         self.schema_mgr = SchemaManager.instance()
         if not self.schema_mgr.is_valid_namespace(namespace):
-            raise ValueError("Namespace '{}' is not in the list of valid namespaces: {}".
-                             format(namespace, self.schema_mgr.get_namespaces()))
+            raise FileNotFoundError("Namespace '{}' is not in the list of valid namespaces: {}".
```

Comment:
I believe this should still be ValueError, as this is a generic store and not specific to FileMetadataStore.

Reply:
ok